docs/dev-setup.md

```diff
@@ -184,10 +184,11 @@ Then download the appropriate `BrowserStackLocal`, unpack it.
 
 When all is set up, run the remote tests:
 1. Run `BrowserStackLocal` with your key: ./BrowserStackLocal --key mYbRoWsErStAcKkEy
-1. Run the tests: ```
-$ export TEST_ENV=remote
-$ yarn run test-e2e
-```
+1. Run the tests:
```

Comment:
Relying on MD to automatically number the lists is a nice concept, but does not always work in practice when there is complex formatting in between. I suggest to start doing explicitly:
```
1.
2.
3.
...
```
instead of:
```
1.
1.
1.
```

Reply:
I noticed the markdown problem today and i was going to open a PR for that, then i saw this one. There is no need for explicit numbering. Problem with auto numbering is indentation. 

> To add another element in a list while preserving the continuity of the list, indent the element four spaces or one tab.
```
1. first
1. second
    ```
    something in between
    ```
1. third
```
is rendered as;

1. first
1. second
    ```
    something in between
    ```
1. third

https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists
https://www.markdownguide.org/basic-syntax/#code-blocks-1

Reply:
I noticed the markdown problem today and i was going to open a PR for that, then i saw this one. There is no need for explicit numbering. Problem with auto numbering is indentation. 

> To add another element in a list while preserving the continuity of the list, indent the element four spaces or one tab.
```
1. first
1. second
    ```
    something in between
    ```
1. third
```
is rendered as;

1. first
1. second
    ```
    something in between
    ```
1. third

https://www.markdownguide.org/basic-syntax/#adding-elements-in-lists
https://www.markdownguide.org/basic-syntax/#code-blocks-1